mad-dog/src/tests/test_integration.js

```diff
@@ -176,6 +186,84 @@ module.exports = coreIntegration = async ({
         emit(Event.RESPONSE, res)
         break
       }
+      case OPERATION_TYPE.CREATE_PLAYLIST: {
+        try {
+          // create playlist
+          const randomPlaylistName = genRandomString(8)
+          const playlist = await executeOne(walletIndex, l =>
+            createPlaylist(l, userId, randomPlaylistName, false, false, [])
```

Comment:
Similar comment here, worth logging what user created a playlist resulting in which ID

Reply:
Will add to the next PR!